config/nav.yml

```diff
@@ -106,6 +106,7 @@ nav:
               - SinkBinding reference: developer/eventing/sources/sinkbinding/reference.md
           - Event delivery: developer/eventing/event-delivery.md
           - Event sinks:
+            - Writing an event source using Golang: developer/eventing/sinks/creating-event-sinks/writing-event-sinks-with-golang/README.md
```

Comment:
```suggestion
            - Writing an event sink using Golang: developer/eventing/sinks/creating-event-sinks/writing-event-sinks-with-golang/README.md
```

Reply:
I think this should also go below Kafka - "About sinks" should stay the first topic in the list, please

Reply:
fixed : https://github.com/knative/docs/pull/4208/commits/972319f71e0fdb34a8f18f1ab41ce013e72d4c53

Thank you!